README.md

```diff
@@ -1,57 +1,205 @@
-A demo app is included to show how to use the project.
+### Step 1: Prepare the app service
 
-To run the demo, follow these steps:
+This guide is for deploying to **Azure App Service** via **VS Code Azure Tools Extension**. Follow these steps with your VSCode workspace set to your copy of the [Enable your Python Django webapp to sign in users and call Microsoft Graph with the Microsoft identity platform](https://github.com/azure-samples/ms-identity-python-django-webapp-call-graph).
 
-(Add steps to start up the demo)
+1. Open the VSCode command palette (ctrl+shift+P on Windows and command+shift+P on Mac).
+1. Choose  `Azure App Service: Create New Web App... (Advanced)`.
+   1. Enter a globally unique name for your web app (e.g. `example-domain`) and press enter. Make a note of this name.
+   2. Click `+ Create new resource group` and choose a name for it (e.g. `example-domain-rg`). Press enter. Make a note of this name.
+   3. Select `Python 3.8` for your runtime stack.
+   4. Click `+ Create new App Service plan` and give it a name (e.g. `example-domain-app-svc-plan`). Press enter.
+   5. Choose an app service tier (e.g., `F1 Free`).
+   6. Click `Skip for now` for application insights resource.
+   7. Select a location (e.g. `West US`). Make a note of this as `westus`.
 
-1.
-2.
-3.
+### Step 2: Prepare the PostgreSQL database
 
-## Resources
+1. See the `azure_settings_example.py` file in the `deployment` folder at the root of the `Enable your Python Django webapp to sign in users and call Microsoft Graph with the Microsoft identity platform` sample. Make a copy of this file, calling it, for example, `my_azure_settings.py`. Don't check this copy in to source control, as it will contain secrets.
+2. Fill in the following details in it:
+   1. APP_SERVICE_APP_NAME from [Step 1: Prepare the app service](#step-1-prepare-the-app-service) 2.a, e.g., **`example-domain`** (**not** `example-domain.azurewebsites.net`)
+   2. AZ_RESOURCE_GROUP from [Step 1: Prepare the app service](#step-1-prepare-the-app-service) 2.b, e.g., `example-domain-rg`
+   3. AZ_LOCATION from [Step 1: Prepare the app service](#step-1-prepare-the-app-service) 2.g, e.g., `westus`
+3. Next, also fill in the following configuration values in your `my_azure_settings.py` file. These values will be used by automation scripts to create the Postgres DB that your deployed app will connect to.
 
-(Any additional resources or related projects)
+   ```Shell
+    POSTGRES_SERVER_NAME='choose a globally-unique name for your server, e.g. my-postgres-server'
+    POSTGRES_ADMIN_USER='choose an admin user name, e.g. administrator'
+    POSTGRES_ADMIN_PASSWORD='choose a password'
+    APP_DB_NAME='choose db name for your app, e.g. my-django-db'
+   ```
 
-- Link to supporting information
-- Link to similar sample
-- ...
+4. From the terminal, run the `create-db.py` script using the command `python deployment/create-db.py`. Answer all of the questions in the affirmative if it is your first run.
```

Comment:
I was getting 403 Forbidden here . `az login --tenant MY_TENANT_ID` resolved my issue here

Reply:
Thanks!
1. Added Azure CLI to requirements
1. Added steps to login to the correct tenant with Azure CLI